pkg/model/dev.go

```diff
@@ -731,16 +733,20 @@ func (dev *Dev) ToTranslationRule(main *Dev) *TranslationRule {
 	return rule
 }
 
-//UpdateNamespace updates the dev namespace
-func (dev *Dev) UpdateNamespace(namespace string) error {
-	if namespace == "" {
-		return nil
+//UpdateContext updates the dev namespace and context
+func (dev *Dev) UpdateContext(namespace, k8sContext string) {
```

Comment:
Can we call this LoadContext or SetContext instead? Update sounds as if we are writing something in the context or changing it.